unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -137,13 +137,15 @@ import Unison.Var (Var)
 import qualified Unison.Var as Var
 import qualified Unison.WatchKind as WK
 import qualified Unison.Codebase.Editor.HandleInput.LoopState as LoopState
-import Unison.Codebase.Editor.HandleInput.LoopState (Action, Action', eval)
+import Unison.Codebase.Editor.HandleInput.LoopState (Action, Action', eval, MonadCommand(..))
 import qualified Unison.Codebase.Editor.HandleInput.NamespaceDependencies as NamespaceDependencies
+import Unison.Util.Free (Free)
+import UnliftIO (MonadUnliftIO)
 
 defaultPatchNameSegment :: NameSegment
 defaultPatchNameSegment = "patch"
 
-prettyPrintEnvDecl :: NamesWithHistory -> Action' m v PPE.PrettyPrintEnvDecl
+prettyPrintEnvDecl :: MonadCommand n m i v => NamesWithHistory -> n PPE.PrettyPrintEnvDecl
```

Comment:
I relax the monad type on a lot of definitions so they can be used within the viewRemoteBranch callback.